.env.test

```diff
@@ -0,0 +1,15 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+# Trello keys
+API_KEY=abcderghij123456789
```

Comment:
These values are fine, but note that you could even use values like `API_KEY=test-trello-api-key` to make it super clear that values are set correctly when e.g. running a test in debug mode.